Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -210,7 +219,7 @@ extension Session {
 }
 
 public func authedRequest(to route: Route, session: Session = .mock) -> URLRequest {
-  let request = URLRequest(url: URL(string: url(to: route))!)
+  let request = router.request(for: route, base: URL(string: "http://localhost:8080"))!
```

Comment:
this was necessary to get requests built up fully, with method & post body, etc...